main/main.c

```diff
@@ -84,9 +99,14 @@ void guiTask() {
     disp_drv.buffer = &disp_buf;
     lv_disp_drv_register(&disp_drv);
 
-#if defined CONFIG_LVGL_TFT_DISPLAY_MONOCHROME
-    lv_theme_mono_init(0, NULL);
-    lv_theme_set_current( lv_theme_get_mono() );
+#ifdef CONFIG_LVGL_TFT_DISPLAY_MONOCHROME
+    lv_theme_mono_init(LV_THEME_DEFAULT_COLOR_PRIMARY, LV_THEME_DEFAULT_COLOR_SECONDARY,
+		LV_THEME_DEFAULT_FLAG,
+		LV_THEME_DEFAULT_FONT_SMALL,
+		LV_THEME_DEFAULT_FONT_NORMAL,
+		LV_THEME_DEFAULT_FONT_SUBTITLE,
+		LV_THEME_DEFAULT_FONT_TITLE);
+    lv_theme_set_act( lv_theme_get_act() );
```

Comment:
This doesn't do anything (take a look at the [implementation](https://github.com/lvgl/lvgl/blob/950ac3b9930ed62bf86fe411b94f234ae644da4d/src/lv_themes/lv_theme.c#L42-L54)).

Reply:
Thanks for taking the time to review it, I just saw the `lv_init` implementation and this step (configuring the theme) is already done in there, so I might take this whole block of code away:

https://github.com/lvgl/lv_port_esp32/blob/3f922ee95e11d54fee2f631b0cb32b8f10a1e103/main/main.c#L102-L110

Reply:
Removed in a526922